fmriprep_rodents/patch/interfaces/__init__.py

```diff
@@ -26,3 +39,278 @@ def _run_interface(self, runtime):
                     'No "%s" images found for sub-%s', imtype, self.inputs.subject_id
                 )
         return runtime
+
+
+class _TemplateFlowSelectOutputSpec(TFSelectOutputSpec):
+    brain_mask = File(desc="Template's brain mask")
+    t2w_file = File(desc="Template's T2w image")
+
+
+class TemplateFlowSelect(_TFSelect):
```

Comment:
Should we move the nipype interface to the templateflow Python client codebase?

Reply:
Doing so would make the python client fairly heavy (we'd need to add nipype as a dependency), so I'm convinced there.

Ultimately I think we'll need to revise the interface to mirror `templateflow.api.get`.